docs/community_data.md

```diff
@@ -0,0 +1,40 @@
+# Community Data
+
+Community data are curated and placed in the designated directory by administrators or curators. Users can access these datasets while working with the [applications](jupyter_apps.md).
+
+Within each Jupyter Hub session, users may locally modify copies of the community data. However, these modifications are session-specific and do not affect the global datasets. Each time a user starts a new Jupyter Hub session, the original community data will be available in the corresponding directory. If users wish to retain any changes made to the community data, they should transfer the modified files to their personal or allocated storage.
+
+
+## How Do Admins Move the Community Data?
+
+The Community Applications project is a special type of project designed for centralized data sharing. Only authorized administrators can add data to the Community Applications project. You can skip this part if you are a regular user.
+
+To upload community data, admins should follow these steps:
+
+1. **Navigate to the Correct Location:**  
+   Community data uploads must be performed through the path:  
+   `Data Files > Dataset > Community Applications`  
+   [Direct link to the Portal](https://digitalporousmedia.org/workbench/data/tapis/projects/drp.project.root)
+
+2. **Initiate the Upload:**  
+   Click the purple **"+Add"** button and select the appropriate action:<br>
+   \- Create a folder<br>
+   \- Create a new dataset<br>
+   \- Upload data from your local system to the current location
```

Comment:
I think you opted for `<br>` and `\-` instead of Markdown (which does support nested lists), because your indentation is only 3 spaces. Though some Markdown parsers support three-spaces, using 4 (or 2) spaces will achieve more consistent results across Markdown parsers. See #13.

Reply:
If your goal was actually to change the spacing before the list, or to change the style of the bullets, let's use CSS for that.

If my guesses are wrong, please tell me your intent.